src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Services/SyntaxFacts/CSharpSyntaxFacts.cs

```diff
@@ -978,71 +979,6 @@ public bool ContainsInMemberBody(SyntaxNode node, TextSpan span)
         private static TextSpan GetBlockBodySpan(BlockSyntax body)
             => TextSpan.FromBounds(body.OpenBraceToken.Span.End, body.CloseBraceToken.SpanStart);
 
-        public int GetMethodLevelMemberId(SyntaxNode root, SyntaxNode node)
```

Comment:
these members were errantly placed in ISyntaxFacts.  i moved them to a dedicated service for semanticmodel reuse as that's all they're intended for.